test/test_files/tinysnb/function/string.test

```diff
@@ -144,6 +144,22 @@ False
 ---- 1
 Carol
 
+-NAME RegExprString
+-QUERY MATCH (a:person) WHERE a.fName =~ 'Ali.*' RETURN a.ID
+---- 1
+0
+
+-NAME RegExprAnchorString
+-QUERY MATCH (a:person) UNWIND a.usedNames as x WITH x AS name WHERE name =~ '^A.*$' RETURN name
+---- 2
+Ad
+Aida
+
+-NAME RegExprUTF8String
+-QUERY MATCH (m:movies) WHERE m.name =~ '.*â.*' RETURN m.name
+---- 1
+Sóló cón tu párejâ
+
```

Comment:
I think it's better to add a bit more expressions for tests here. e.g., the one with escapes.

Reply:
added 3 more tests with escape sequence regex